Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.AutoImplementProperties.Attributes.CSharp/Csla.Generator.AutoImplementProperties.Attributes.CSharp.csproj

```diff
@@ -18,6 +18,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Backport.System.Threading.Lock" Version="3.1.0" />
+    <PackageReference Include="Backport.System.Threading.Lock" Version="3.1.4" />
+    <Using Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))" Alias="Lock" Include="System.Threading.Lock" />
+    <Using Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))" Alias="Lock" Include="Backport.System.Threading.Lock" />
```

Comment:
At this point, wouldn't it make more sense to make the `Lock` alias simply refer to `System.Object`? This is assuming that the only way the `Lock` type is currently being used is through the `lock (...)` syntax and not through any method calls. That way the external dependency can be dropped in its entirety.

Reply:
That would not allow the new locking API though. If an external dependency is an issue it can be used as the source generator instead.

Reply:
See my other comment regarding usage of the `Backport` package - it is only used in a couple projects, and so no other projects (including the generators) should reference the package at all right?